packages/allow-deny/allow-deny.js

```diff
@@ -0,0 +1,493 @@
+///
+/// Remote methods and access control.
+///
+
+// Restrict default mutators on collection. allow() and deny() take the
+// same options:
+//
+// options.insert {Function(userId, doc)}
+//   return true to allow/deny adding this document
+//
+// options.update {Function(userId, docs, fields, modifier)}
+//   return true to allow/deny updating these documents.
+//   `fields` is passed as an array of fields that are to be modified
+//
+// options.remove {Function(userId, docs)}
+//   return true to allow/deny removing these documents
+//
+// options.fetch {Array}
+//   Fields to fetch for these validators. If any call to allow or deny
+//   does not have this option then all fields are loaded.
+//
+// allow and deny can be called multiple times. The validators are
+// evaluated as follows:
+// - If neither deny() nor allow() has been called on the collection,
+//   then the request is allowed if and only if the "insecure" smart
+//   package is in use.
+// - Otherwise, if any deny() function returns true, the request is denied.
+// - Otherwise, if any allow() function returns true, the request is allowed.
+// - Otherwise, the request is denied.
+//
+// Meteor may call your deny() and allow() functions in any order, and may not
+// call all of them if it is able to make a decision without calling them all
+// (so don't include side effects).
+
+AllowDeny = {
+  CollectionPrototype: {}
+};
+
+// In the `mongo` package, we will extend Mongo.Collection.prototype with these
+// methods
+const CollectionPrototype = AllowDeny.CollectionPrototype;
+
+/**
+ * @summary Allow users to write directly to this collection from client code, subject to limitations you define.
+ * @locus Server
+ * @method allow
+ * @memberOf Mongo.Collection
+ * @instance
+ * @param {Object} options
+ * @param {Function} options.insert,update,remove Functions that look at a proposed modification to the database and return true if it should be allowed.
+ * @param {String[]} options.fetch Optional performance enhancement. Limits the fields that will be fetched from the database for inspection by your `update` and `remove` functions.
+ * @param {Function} options.transform Overrides `transform` on the  [`Collection`](#collections).  Pass `null` to disable transformation.
+ */
+CollectionPrototype.allow = function(options) {
+  addValidator(this, 'allow', options);
+};
+
+/**
+ * @summary Override `allow` rules.
```

Comment:
I don't know if you're looking for feedback on the docs, but I don't understand the description of `deny` as "override `allow` rules". Or maybe there is an additional sentence missing or something?